dags/iop/parser.py

```diff
@@ -39,6 +40,12 @@ def __init__(self) -> None:
                 destination="arxiv_eprints",
                 extraction_function=self._get_extracted_arxiv_eprint_value,
             ),
+            AttributeExtractor(
+                destination="page_nr",
+                source="front/article-meta/counts/page-count",
+                attribute="count",
+                extra_function=lambda x: parse_to_int(x, self.logger),
```

Comment:
you don't need to pass around the logger :) it's module could have its own logger.

Reply:
should I just print it?